mne/viz/_3d.py

```diff
@@ -593,6 +598,16 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
 
     if isinstance(surfaces, str):
         surfaces = [surfaces]
+    if isinstance(surfaces, dict):
+        user_alpha = surfaces
```

Comment:
Dicts are mutable and we modify this later so we need to copy so we don't change the user's variable
```suggestion
        user_alpha = surfaces.copy()
```

Reply:
I figured since surfaces is reassigned in the next lines it wouldn't matter. But agreed this is safer/cleaner. Adding!